pkgdb/src/buildenv/realise.cc

```diff
@@ -662,47 +660,53 @@ addScriptToScriptsDir( const std::string &           scriptContents,
   std::filesystem::permissions( scriptPath,
                                 std::filesystem::perms::owner_exec,
                                 std::filesystem::perm_options::add );
+}
 
-  /* Reference the script from the main activation script. */
-  auto scriptRelativePath
-    = nix::fmt( "\"$FLOX_ENV/%s/%s\"", ACTIVATION_SUBDIR_NAME, scriptName );
-  if ( shouldSource )
-    {
-      mainActivationScriptContents << "source " << scriptRelativePath << '\n';
-    }
-  else
-    {
-      mainActivationScriptContents << FLOX_BASH_BIN << " " << scriptRelativePath
-                                   << '\n';
-    }
+std::string
+activationScriptRelativePath( const std::string & scriptName )
+{
+  return nix::fmt( "\"$FLOX_ENV/%s/%s\"", ACTIVATION_SUBDIR_NAME, scriptName );
```

Comment:
This is the way it was done beforehand, I just extracted it into a helper function

Reply:
Isn't this producing an absolute path though, given that `FLOX_ENV` is an absolute path?
Granted it's relative to that 🤷🏼‍♂️

Reply:
Yeah I meant as opposed to a /nix/store path

Reply:
I updated the symbol name to `activationScriptEnvironmentPath` to help disambiguate what it's doing.  As far as why then environment vs `/nix/store`.  That I'm not sure.   Is there a concern with doing it this way or an advantage to having it in the store?

Reply:
It's more of a nit, definitely nonblocking. Because we have a `/nix/store` path that will never change here, it seems pointless and more fragile to introduce indirection via an environment variable.

Reply:
As far as I can tell... it looks like the final activation store path is identified as a return from `addToStore` when the *temp* path that these scripts are getting created in, is moved into the store.  So as it's coded now we don't know it at the time we are creating the scripts.  To avoid the use of the ENV var, some refactoring would be required... at a minimum.